cli-plugins/apps/create-workload.md

```diff
@@ -15,81 +15,120 @@ The following prerequisites are required to use workloads with Tanzu Application
 
 ## <a id="example"></a> Get started with an example workload
 
-Here is how you can get started with an example workload.
+### <a id="workload-git"></a> Create a workload from GitHub repository
 
-- To name the workload and specify a source code location to create the workload from, run:
+Tanzu Application Platform supports creating a workload from an existing git repository by setting the flags `--git-repo`, `--git-branch`, `--git-tag` and `--git-commit`, this will allow the out of the box [supply chain](../../scc/about.md) to get the source from the given repository to deploy the application.
 
+To create a named workload and specify a git source code location, run:
+
+ ```console
+tanzu apps workload create pet-clinic --git-repo https://github.com/sample-accelerators/spring-petclinic --git-tag tap-1.1 --type web  
```

Comment:
may need submit a separate PR for TAP 1.1 vs. TAP 1.2 docs updates WRT /cli-plugins/apps/create-workload.md

`--git-tag tap-1.1` (for TAP 1.1)
`--git-tag tap-1.2` (for TAP 1.2)